src/Entity/Slot.php

```diff
@@ -0,0 +1,223 @@
+ *       "delete" = "Drupal\acquia_lift\Form\SlotDeleteConfirmForm",
+ *       "disable" = "Drupal\acquia_lift\Form\SlotDisableConfirmForm",
+ *       "enable" = "Drupal\acquia_lift\Form\SlotEnableConfirmForm",
+ *     },
+ *   },
+ *   admin_permission = "administer acquia_lift",
+ *   config_prefix = "slot",
+ *   entity_keys = {
+ *     "id" = "id",
+ *     "label" = "label",
+ *     "uuid" = "uuid",
+ *     "status" = "status",
+ *   },
+ *   config_export = {
+ *     "id",
+ *     "label",
+ *     "description",
+ *     "visibility",
+ *     "css_url"
+ *   },
+ *   links = {
+ *     "add-form" = "/admin/config/content/acquia-lift/slots/add-slot",
+ *     "edit-form" = "/admin/config/content/acquia-lift/slots/{acquia_lift_slot}",
+ *     "delete-form" = "/admin/config/content/acquia-lift/slots/{acquia_lift_slot}/delete",
+ *     "disable" = "/admin/config/content/acquia-lift/slots/{acquia_lift_slot}/disable",
+ *     "enable" = "/admin/config/content/acquia-lift/slots/{acquia_lift_slot}/enable",
+ *   }
+ * )
+ */
+class Slot extends ConfigEntityBase implements SlotInterface {
+
+  /**
+   * The ID of the slot.
+   *
+   * @var string
+   */
+  protected $id;
+
+  /**
+   * The slot label.
+   *
+   * @var string
+   */
+  protected $label;
+
+  /**
+   * A string describing the slot.
+   *
+   * @var string
+   */
+  protected $description;
+
+  /**
+   * The slot visibility settings
+   *
+   * @var string
+   */
+  protected $visibility;
+
+  /**
+   * The slot CSS Url
+   *
+   * @var string
+   */
+  protected $css_url;
+
+  /**
+   * The Lift API Helper.
+   *
+   * @var \Acquia\LiftClient\Lift
+   */
+  protected $liftClient;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(array $values, $entity_type) {
+    parent::__construct($values, $entity_type);
+
+    try {
+      /** @var \Drupal\acquia_lift\Service\Helper\LiftAPIHelper $liftHelper */
+      $liftHelper = \Drupal::getContainer()
+        ->get('acquia_lift.service.helper.lift_api_helper');
+      $this->liftClient = $liftHelper->getLiftClient();
+    } catch (AcquiaLiftException $e) {
+      drupal_set_message(t($e->getMessage()), 'error');
+    }
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getDescription() {
+    return $this->description;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setDescription($description) {
+    $this->description = $description;
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCssUrl() {
+    return $this->css_url;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setCssUrl($cssUrl) {
+    $this->css_url = $cssUrl;
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getHtml() {
+    $css_html = "";
+    if (!empty($this->css_url)) {
+      $css_html = " data-lift-css=\"" . $this->css_url . "\"";
+    }
+    return "<div data-lift-slot=\"" . $this->id() . "\"" . $css_html . "/>";
+  }
```

Comment:
Personally I'd try to keep the entity dumb and logic-less ... In away that I'd never need to unit test a Entity, but I understand why you'd put it here.